src/app/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+import './app.scss';
+import * as mac from 'assets/imgs/mac.png';
+import * as windows from 'assets/imgs/windows.png';
+import * as linux from 'assets/imgs/linux.png';
+import * as Secure from 'assets/imgs/icon-shield.svg';
+import * as Offline from 'assets/imgs/icon-offline.svg';
+import * as Yours from 'assets/imgs/icon-hdd.svg';
+import * as OpenSource from 'assets/imgs/icon-github.svg';
+import { Block } from './block';
+import { Platform } from './platform';
+
+const builds = [
+  { name: 'Mac', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Windows', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Linux', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'HTML Build', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' }
+];
+
+let OS: string = '';
+if (navigator.appVersion.includes('Win')) {
+  OS = 'Windows';
+} else if (navigator.appVersion.includes('Mac')) {
+  OS = 'Mac';
+} else if (navigator.appVersion.includes('Linux')) {
+  OS = 'Linux';
+} else if (navigator.appVersion.includes('X11')) {
+  OS = 'Unix';
+}
+const img: any = OS === 'Mac' ? mac : OS === 'Windows' ? windows : linux;
+
+const App = () => (
+  <div className="App">
+    <main className="App-main">
+      <div className="App-main-left">
+        <h1 className="App-title">MyCrypto for {OS}</h1>
+        <p className="App-description">
+          Run MyCrypto securely and offline with the desktop application. Powered by Electron, it's
+          the same MyCrypto app, running in a sandboxed environment.
+        </p>
+        <a
+          href="https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC"
+          className="App-download"
+        >
+          <i className="nc-icon nc-hit-down" />
+          Download Alpha
```

Comment:
Opinion: I think `MyCrypto for Desktop` would be a better title, with `Download for {os}` being a better CTA. If we're going to call out the alpha, I think it should be more prominent, but given its own easily removable space for when we launch. Would definitely like to hear more thoughts on this.

Reply:
Simple example, it would be ugly as hell, but drive the point home: We could use the MyCrypto announcement style (danger or warning color) to indicate alpha at the top of the page.

Reply:
About the 'MyCrypto for Desktop', I think it's a bit more compact to have 'MyCrypto for {os}' with a more actionable cta button. We already customize the screenshots for each os, so it would make sense to do that in the title too. Also, I think a download button is enough to indicate to the user they can download the app, no need for additional text, especially when we already have a bit of copy there to describe the app.